counter/src/main/java/edu/nextstep/camp/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,33 @@
+package edu.nextstep.camp.counter
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel(initialValue: Int? = null) : ViewModel() {
```

Comment:
```suggestion
class CounterViewModel(initialValue: Int = 0) : ViewModel() {
```
생성자에는 nullable이지만, 실제로 내부에서 사용하는 값은 null일 경우 0으로 초기화하고 있네요.
초깃값을 null이 아닌 0으로 설정해보면 어떨까요?